package.json

```
@@ -20,10 +20,10 @@
     "test": "mocha"
   },
   "dependencies": {
-    "drafter.js": "^3.0.0-pre.2"
+    "drafter.js": "^3.0.0"
```

I am thinking that maybe fury-adapter-apib-parser (drafter-npm) should pin to exact parsers so that consumers of fury can pin the adapter to exact parser. It's not possible, for example for "dredd" to pin to one specific parser. /c @honzajavorek 

```suggestion
    "drafter.js": "3.0.0"
```

---

I don't often find myself wanting to pin to a specific parser version. I think Dredd's major issue is that it is not able to choose whether to compile the parser or not, i.e. it is not able to choose the parser library used down the fury toolchain.

---

Ok, but you do pin to specific versions of the OAS 2 and OAS 3 parsers in dredd-transactions. Is there specific reason for not making them a range like API Blueprint parser ends up (via fury-adapter-apib-parser)?

---

I have all dependencies in all repositories pinned. I like the fact that the upgrades are explicit by dependabot (previously greenkeeper) and I see the CI running for each such change. With ranges you have to rely on people not introducing breaking changes to minor/patch releases, but humans are mess and they do mistakes, so that's fragile. Lock files could help, but only in case of applications (as opposed to libraries) and @apiaryio/sjs has almost none, everything we do is libraries where the lock file is mostly for us, not for our users.